static/files/forms-data.json

```diff
@@ -3,80 +3,24 @@
     "/data/postgresql": {
       "templatePath": "/data/postgresql/index.html",
       "isModal": true,
-      "modalId": "contact-modal",
+      "modalId": "data-relational-dbs-modal",
       "formData": {
-        "title": "Test form",
-        "introText": "Intro text",
-        "formId": "1266",
+        "title": "Talk to our relational databases' experts",
+        "formId": "5477",
         "returnUrl": "/data/postgresql#contact-form-success",
         "product": ""
       },
       "fieldsets": [
         {
-          "title": "About your company",
-          "id": "about-company",
-          "noCommentsFromLead": true,
-          "fields": [
-            {
-              "type": "text",
-              "id": "company",
-              "label": "Company name",
-              "isRequired": true
-            },
-            {
-              "type": "text",
-              "id": "title",
-              "label": "Job title",
-              "isRequired": true
-            }
-          ]
-        },
-        {
-          "title": "What would you like to talk to us about?",
+          "title": "Tell us more about your use case",
           "id": "comments",
-          "isRequired": true,
+          "isRequired": false,
           "fields": [
             {
               "type": "long-text",
-              "id": "comments"
-            }
-          ]
-        },
-        {
-          "title": "How many devices?",
-          "id": "how-many-machines",
-          "isRequired": true,
-          "inputName": "how-many-machines-do-you-have",
-          "fields": [
-            {
-              "type": "radio",
-              "id": "less-5-machines",
-              "value": "less than 5",
-              "label": "< 5 machines"
-            },
-            {
-              "type": "radio",
-              "id": "5-to-15-machines",
-              "value": "5 to 15 machines",
-              "label": "5 - 15 machines"
-            },
-            {
-              "type": "radio",
-              "id": "15-to-50-machines",
-              "value": "15 to 50 machines",
-              "label": "15 - 50 machines"
-            },
-            {
-              "type": "radio",
-              "id": "50-to-100-machines",
-              "value": "50 to 100 machines",
-              "label": "50 - 100 machines"
-            },
-            {
-              "type": "radio",
-              "id": "greater-than-100",
-              "value": "greater than 100",
-              "label": "> 100 machines"
+              "id": "comments",
+              "ariaLabel": "Anything you'd like to communicate about your needs or interests?",
+              "placeholder": "Anything you'd like to communicate about your needs or interests?"
```

Comment:
A screen reader will read the placeholder, so as they have the same content you can omit the `aria-label`